.github/workflows/render-readme.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: macOS-latest
```

Comment:
might be better (cheaper) to run this on ubuntu

Reply:
I believe it's all free since the package is public